features/gift_exchanges/challenge_yuletide.feature

```diff
@@ -262,7 +267,8 @@ Feature: Collection
   Then I should see "Sign-up was successfully created"
 
   # ordinary users can't see signups until 5 people have signed up
-  When I go to the collections page
+  When all indexing jobs have been run 
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.